### DISCUSSION TITLE
Can't set "binary prefix" setting

### DISCUSSION THREAD

**Discussion by Xotchkass** [Qalculate/libqalculate#590] [Q&A] [answered]
Despite setting it to `on` it still doesn't print prefix in output. 
![изображение](https://github.com/Qalculate/libqalculate/assets/78706300/7fcc5a30-65b2-44f2-bbca-88c496c42ad8)

**Comment by hanna-kn** [comment, marked as answer]
```
> help binary prefixes

binary prefixes (binpref)
If activated, binary prefixes are used by default for information units.
(0* = off, 1 = on)

> 1024byte

  1024 bytes = 1.024 kB

> set binary prefixes

  1.024 kB = 1 KiB

```

Try `set base display 2` instead.
```
> help base display

base display (basedisp)
(0 = none, 1* = normal, 2 = alternative)

> set basedisp 2
> 42->bin

  42 = 0b0000101010

```

**Reply by Xotchkass** [reply]
Oh. Thanks.

**Reply by Xotchkass** [reply]
Ok, Another question. Will ask here to not create a new thread. How can I set it persistently?
I usually use qalc as cli command and not interactively, but `-set "base display 2"` just launches interactive mode and after quitting and running from terminal it just sets to defaults.
![изображение](https://github.com/Qalculate/libqalculate/assets/78706300/b9bf06c8-8b14-47b7-acda-33d0f6041242)

**Reply by hanna-kn** [reply]
The settings (including changes made with command-line options) should normally be saved when exiting from interactive mode, unless using Ctrl+C (use Ctrl+D or quit/exit command on Windows). Settings can also be saved using `save mode` command.

**Reply by Xotchkass** [reply]
Oh, This is exactly how I was closing it. Thanks again.
